electron/execution.ts

```diff
@@ -56,7 +59,7 @@ async function launchContext() {
   }
 
   context.on('page', page => {
-    page.on('dialog', () => {});
+    // page.on('dialog', () => {});
```

Comment:
I don't know what it does but TS complains as the callback is empty. Can we remove this listner on 'dialog' event?

Reply:
I was able to record a journey and e2e tests are passing, so I think no problems here.

Reply:
Not sure why this is here in the first place 😆